src/test/java/EthereumTest.java

```diff
@@ -0,0 +1,59 @@
+import org.bouncycastle.math.ec.custom.sec.SecP256K1Curve;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.web3j.crypto.Bip32ECKeyPair;
+import org.web3j.crypto.ECKeyPair;
+
+import java.util.HexFormat;
+
+public class EthereumTest {
```

Comment:
Presumably a person trying to do this would be a web3j rookie I think maybe using randomly created keys would also have value rather than magic values from an example standpoint.

Reply:
Also if it can use a BIP39 seed phrase that would be ideal. If the library doesn't support that generation maybe just generate one online and magic value the seed phrase in with a bit of an explanation.